backend/connection.ts

```diff
@@ -1,49 +1,58 @@
-const dotenv = require('dotenv')
-const mongoose = require('mongoose')
+const dotenv = require("dotenv");
+const mongoose = require("mongoose");
 
-dotenv.config()
+dotenv.config();
 
 type ErrorWithMessage = {
-   message: string
- }
-
-function makeNewConnection(url : string) {
-    const db = mongoose.createConnection(url, {
-        useNewUrlParser: true,
-        useUnifiedTopology: true,
-    });
-    // https://kentcdodds.com/blog/get-a-catch-block-error-message-with-typescript
-    function isErrorWithMessage(error: unknown): error is ErrorWithMessage {
-      return (
-        typeof error === 'object' &&
-        error !== null &&
-        'message' in error &&
-        typeof (error as Record<string, unknown>).message === 'string'
-      )
+  message: string;
+};
+
+function makeNewConnection(url: string) {
+  const db = mongoose.createConnection(url, {
+    useNewUrlParser: true,
+    useUnifiedTopology: true,
+  });
+  // https://kentcdodds.com/blog/get-a-catch-block-error-message-with-typescript
+  function isErrorWithMessage(error: unknown): error is ErrorWithMessage {
+    return (
+      typeof error === "object" &&
+      error !== null &&
+      "message" in error &&
+      typeof (error as Record<string, unknown>).message === "string"
+    );
+  }
+
+  db.on("error", function (error: unknown) {
+    if (isErrorWithMessage(error)) return error;
+    try {
+      console.log(`MongoDB :: connection ${JSON.stringify(error)}`);
+      db.close().catch(() =>
+        console.log(`MongoDB :: failed to close connection`)
+      );
+    } catch {
+      return new Error(String(error));
     }
+  });
 
-    db.on('error', function (error: unknown) {
-      if (isErrorWithMessage(error)) return error
-      try {
-         console.log(`MongoDB :: connection ${JSON.stringify(error)}`);
-         db.close().catch(() => console.log(`MongoDB :: failed to close connection`));
-       } catch {
-         return new Error(String(error))
-       }
-    });
-
-    db.on('connected', function () {
-      console.log(`MongoDB :: connected`);
+  db.on("connected", function () {
+    console.log(`MongoDB :: connected`);
   });
-  
-  db.on('disconnected', function () {
-      console.log(`MongoDB :: disconnected`);
+
+  db.on("disconnected", function () {
+    console.log(`MongoDB :: disconnected`);
   });
-  
-    return db;
+
+  return db;
 }
 
-const userConnection = makeNewConnection(process.env.userDB!)
-const gardenConnection = makeNewConnection(process.env.gardenDB!)
+const userConnection = makeNewConnection(process.env.userDB!);
+const gardenConnection = makeNewConnection(process.env.gardenDB!);
 
-export { userConnection, gardenConnection };
\ No newline at end of file
+<<<<<<< HEAD
+module.exports = {
+  userConnection,
+  gardenConnection,
+};
+=======
+export { userConnection, gardenConnection };
+>>>>>>> d6d141676ac82f1d56920b8be5ba288a3422b9dc
```

Comment:
it looks like when you were resolving the merge conflicts you left `<<<<<<<< Head` and similar lines in some of these files, remove them and make another push